cmd/placement/config.go

```diff
@@ -61,8 +59,7 @@ func newConfig() *config {
 
 	flag.StringVar(&cfg.raftID, "id", cfg.raftID, "Placement server ID.")
 	flag.StringVar(&cfg.raftPeerString, "initial-cluster", cfg.raftPeerString, "raft cluster peers")
-	flag.BoolVar(&cfg.raftBootStrap, "cluster-bootstrap", cfg.raftBootStrap, "raft cluster bootstrap required")
-	flag.BoolVar(&cfg.raftInMemEnabled, "inmem-store-enabled", cfg.raftInMemEnabled, "Enable in-memory log and snapshot store")
+	flag.BoolVar(&cfg.raftInMemEnabled, "inmem-store-enabled", cfg.raftInMemEnabled, "Enable in-memory log and snapshot store unless --raft-logstore-path is set")
```

Comment:
What's the default here? Also, is the default our production recommendation? if so, should this be documented?

Reply:
default one is `true`. without any flag, it will use memory storage to save log and snapshopts.  document must be done before release.

Reply:
So without HA mode, placement will save logs and snapshot in memory. with HA mode, placement will be 3 instances and save logs and snapshot in persistent volume.